src/history/historyeventoptions.ts

```diff
@@ -2,7 +2,7 @@
 export class HistoryEventOptions {
     public details?: string;
     public fireAt?: Date;
-    public input?: string;
+    public input?: string | null;
```

Comment:
Elsewhere you use `undefined` instead of `null`. Why is this one different?

Reply:
Good catch. The general recommended pattern in TypeScript is to try to stick to `undefined`, as that limits the number of checks you need to make.